### PR TITLE
Fix TAO 10122 - IE11 graphic gap interaction when navigate to question since response

### PR DESCRIPTION
**Pull request by habemuscode**
**Related to**

https://oat-sa.atlassian.net/browse/TAO-10122

**Description**

InternetExplorer: Have created a Graphic Gap interactions with images and matched them in response
When I do different actions with functionality, at some point, I can't seem to edit the size of the "Gap match slots" 
I am not experiencing the same problems in f.eks Chrome

Workaround: Click outside the interaction and try again

**How to test it**

- Add graphic 'Gap match' interaction to the canvas in the Internet Explorer browser
- Select a file and click 'Select'
- Click the "Plus in circle" icon
- Select `file` and click 'Select' 
- Click 'Response' and set the added picture to the gap 
-  Click 'Question' and find media sizer in the Interaction properties panel

**Experienced Behavior**

The interaction properties panel is missing.

**Expected Behavior**

The interaction properties panel is displayed as soon as 'Question' is clicked in interaction.

### CODE REVIEW

File: src/interactUtils.js [hunk 1]
```diff
@@ -24,6 +24,7 @@ import $ from 'jquery';
 import _ from 'lodash';
 import interact from 'interact';
 import 'core/mouseEvent';
+import triggerCustomEvent from '@oat-sa/tao-core-sdk/src/core/customEvent';
```
Comment (wazelin) [inline src/interactUtils.js:27]: Shouldn't the SDK dependency be specified as a production dependency in `package.json`?

Comment (habemuscode) [reply src/interactUtils.js]: No longer needed. Thanks for comment

File: src/interactUtils.js [hunk 2]
```diff
@@ -74,18 +74,28 @@ interactHelper = {
...
                 height: 100,
                 isPrimary: true
             };
         if (element) {
             domElement = element instanceof $ ? element.get(0) : element;
 
-            domElement.dispatchEvent(new PointerEvent('pointerdown', eventOptions));
-            domElement.dispatchEvent(new PointerEvent('pointerup', eventOptions));
+            var firstEvent;
+            var secondEvent;
```
Comment (wazelin) [inline src/interactUtils.js:87]: `var` variable declarations SHOULD be on the top of the scope.

Please either transform the whole thing into ES6, or move them upper.

Comment (habemuscode) [reply src/interactUtils.js]: Done

**Pull request merged**